core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/ui/dot-experiments-details-table/dot-experiments-details-table.component.html

```diff
@@ -1,53 +1,81 @@
-<div class="flex flex-column gap-2">
-    <div
-        class="header-title flex flex-row flex-wrap align-items-center"
-        *ngIf="title"
-        data-testId="header-title"
-    >
-        <ng-container *dotStringTemplateOutlet="title">
-            {{ title }}
-        </ng-container>
-    </div>
-
-    <div class="flex flex-column gap-3 details">
-        <div class="details__headers grid grid-nogutter">
-            <ng-container
-                *ngTemplateOutlet="headers || defaultHeaders; context: { $implicit: data }"
-            ></ng-container>
+<ng-container *ngIf="!loading; else loadingStateTpl">
+    <div class="flex flex-column gap-2" *ngIf="!empty; else emptyStateTpl">
```

Comment:
This is exaclty why I keep telling people to read and understansd this: https://kentcdodds.com/blog/stop-using-isloading-booleans

Reply:
In this case, it comes from a selector check if the result (in this case, a successful response) but not enough sessions `results?.sessions.total === 0 ` I can change to `hasEnoughSessions` to be more specific